src/main/java/com/example/simulator/data/sensors/TemperatureSimulation.java

```diff
@@ -0,0 +1,40 @@
+package com.example.simulator.data.sensors;
+
+import com.example.simulator.data.DoubleSimulation;
+import lombok.*;
+
+import java.util.Random;
+
+@Data
+public class TemperatureSimulation implements DoubleSimulation {
+    @Getter
+    private Double simulatedValue;
+
+    @Getter
+    @Setter
+    private Double targetValue;
+
+    private Double randomizationValue = 2d;
+    private Double changeSpeed = .1;
+
+    public TemperatureSimulation(Double simulatedValue, Double targetValue) {
+        this.simulatedValue = simulatedValue;
+        this.targetValue = targetValue;
+    }
+
+    public void triggerSimulation() {
+        Random random = new Random();
+
+        var deltaRaw = targetValue - simulatedValue;
+
+        var delta = (deltaRaw * changeSpeed) + ((random.nextDouble() - .5) * randomizationValue);
```

Comment:
Czy to na pewno działa? Z tego co widzę, jeśli:
- `targetValue = 25.0`,
- `simulatedValue = 20.0`,

to w pierwszej iteracji `deltaRaw` wyniesie `5.0`, natomiast
to delta będzie wynosić:
`(deltaRaw * changeSpeed) + ((random.nextDouble() - .5) * randomizationValue)`,
czyli
`(5 * 0.1) + (<-0.5, 0.5> * 2)`, gdzie `<-0.5, 0.5>` to liczba z przedziału od -0.5 do 0.5,
czyli
- maksymalnie: 0.5 + 0.5*2 = 0.5+1 = 1.5 --> temperatura wzrośnie o 1.5 stopnia Celsjusza ✅
- minimalnie: 0.5 + (-0.5)*2 = 0.5 - 1 = -0.5 --> temperatura zmaleje o 0.5 stopnia Celsjusza ❌

Jeśli będzie drugi przypadek, to w drugiej iteracji może być znowu spadek, tym razem o:
(5.5 * 0.1) + (-0.5 * 2) = 0.55 - 1 = -0.45

W kolejnych iteracjach znowu może być problem.

@julek64 prosiłbym o wyjaśnienie, bo może to źle zinterpretowałem, ale jak odpaliłem ten kod tych danych, to miałem taki wynik:
```
delta: 0.8545672708287955
simulatedValue: 20.9

delta: -0.06991377069321336
simulatedValue: 20.799999999999997

delta: 0.02560322663310649
simulatedValue: 20.799999999999997

delta: 0.24202988254822383
simulatedValue: 20.999999999999996

delta: -0.4393331512382177
simulatedValue: 20.599999999999998

delta: 1.2377208314545853
simulatedValue: 21.799999999999997

delta: -0.6101729864233831
simulatedValue: 21.199999999999996

delta: 0.3365032748234661
simulatedValue: 21.499999999999996

delta: -0.09601832594060816
simulatedValue: 21.399999999999995

delta: -0.6227941069884948
simulatedValue: 20.799999999999994
```
czyli widać, że temperatura raz rośnie, raz maleje, co raczej nie jest pożądanym efektem.